docs/topics/mta-rn-resolved-issues-7-0-3.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes-7.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-7-0-3_{context}"]
+= Resolved issues
+
+This section provides issues that have been resolved in {ProductFullName} version 7.0.3.
+
+.Fixed incorrect assessment status when running an assessment on two questionnaires 
+
+In {ProductShortName} 7.0.2, running two questionnaires displayed the `Assessment` status as `Not started` instead of `In progress`. With this update, the problem has been fixed. As a result, the Assessment status shows `In progress` after one questionnaire or archetype is started. list:https://issues.redhat.com/browse/MTA-2409[(MTA-2409)] list:[(MTA-2410,] list:https://issues.redhat.com/browse/MTA-2410[MTA-2410)]
```

Comment:
```suggestion
In {ProductShortName} 7.0.2, running two questionnaires displayed the `Assessment` status as `Not started` instead of `In progress`. With this update, the problem has been fixed. As a result, the Assessment status shows `In progress` after one questionnaire or archetype is started. 

* link:https://issues.redhat.com/browse/MTA-2409[(MTA-2409)] 

* link:https://issues.redhat.com/browse/MTA-2410[MTA-2410)]
```